### DISCUSSION TITLE
GitBook Private Beta

### DISCUSSION THREAD

**Discussion by addisonschultz** [GitbookIO/community#364] [Announcements]
[Edited March 20th 2024]

The mentioned features and integrations are no longer in beta - this post is now outdated! 

---

Hello everyone!

Addison here from GitBook. We have some exciting news around a few features we've been working on. The last few weeks we've been exploring new ways you and your team can work together more efficiently, and testing out some integrations and workflows that will allow you to share knowledge faster and easier with your organization — Including major tools you already use like **GitHub**, **Slack** and **VS Code**.

We've just opened up our private beta, and wanted to bring GitBookers like you in on the action.

Feel free to join us over at [www.gitbook.com/beta](https://www.gitbook.com/beta) to learn more.

Oh, and an added bonus? You'll get a sneak peak at some new designs and branding we've been hard at work on.

See you there! 🚀

**Comment by zhimaj** [comment]
Whether to enter the test version or not is the official version
